qt-models/weightsysteminfomodel.cpp

```diff
@@ -98,7 +98,7 @@ void WSInfoModel::updateInfo()
 	beginRemoveRows(QModelIndex(), 0, this->rows);
```

Comment:
Here and elsewhere: To reset the model, there is `beginResetModel()`/`endResetModel()`.

Reply:
yes, this can work.